.circleci/config.yml

```diff
@@ -135,32 +161,29 @@ workflows:
       - spawn-stability-tests-job:
           requires:
             - lint
-            - test
+            - unit-tests
+            - integration-tests
             - build
           filters:
             branches:
               only: /master/
             tags:
               ignore: /.*/
+      - integration-tests:
+          filters:
+            tags:
+              only: /.*/
```

Comment:
Does this mean it'll run on all branches *and* all tags?

Reply:
I just copied from unit tests. But yes looks like that's the case.